### ISSUE TITLE
Template kann in /templates/unterverzeichnis-templates nicht gefunden werden

### ISSUE THREAD

**Issue by Olli** [heimrichhannot/contao-filter-bundle#5]
Contao 4.9.8 - Filterbundle 1.5.4

Ich habe ein Template für mein Formular in /templates/unterverzeichnis abgelegt. In der Filterconfig ist dieses Template auch auswählbar. 
Ich bekomme aber einen Fehler, dass Template nicht findbar ist.
Folgende Fehlermeldung kommt

`request.CRITICAL: Uncaught PHP Exception Twig\Error\LoaderError: "Unable to find template "/www/htdocs/*/*/templates/unterverzeichnis-templates/filter_form_*_div_layout.html.twig" (looked into: /www/htdocs/*/*/vendor/knplabs/knp-menu/src/Knp/Menu/Resources/views, /www/htdocs/*/*/templates, /www/htdocs/*/*/vendor/symfony/twig-bridge/Resources/views/Form)." at /www/htdocs/*/*/vendor/twig/twig/src/Loader/FilesystemLoader.php line 250 {"exception":"[object] (Twig\\Error\\LoaderError(code: 0): Unable to find template \"/www/htdocs/*/*/templates/unterverzeichnis-templates/filter_form_*_div_layout.html.twig\" (looked into: /www/htdocs/*/*/vendor/knplabs/knp-menu/src/Knp/Menu/Resources/views, /www/htdocs/*/*/templates, /www/htdocs/*/*/vendor/symfony/twig-bridge/Resources/views/Form). at /www/htdocs/*/*/vendor/twig/twig/src/Loader/FilesystemLoader.php:250)"}`

So wie es aussieht wird in /templates gesucht aber nicht in /templates/unterverzeichnis-templates .
Das Verzeichnis ist aber im Theme als templates Ordner angegeben.

**Comment by Defcon0** [comment]
@koertho Korrigiere mich, wenn falsch, aber meines Wissens ist das Core-Verhalten. Contao unterstützt auch keine Unterordner im templates-Ordner, wenn das Template heißt wie eines im Core oder einem anderen Bundle. Erst, wenn es einen eigenen Namen hat, geht es.

**Comment by Olli** [comment]
Ja da muss ich dich leider korrigieren. Du kannst, sofern du den Unterordner bei den Templateeinstellungen verbunden hast ein ce_text.html5 rein legen und es funktioniert und wird in der Templateauswahl von ce_text als ce_text (mein Template) angezeigt.

filter_form_*_div_layout.html.twig ← ist customized ... der * ist der Kundenname ... ich habe das einfach mit suchen/ersetzen gegen * ersetzt was ich nicht auf Github hochladen möchte.

**Comment by Defcon0** [comment]
Ja, das ist mir schon klar. Ich meinte eher Unterordner von Theme-Ordnern, d.h.

`<root>/templates/my-theme/another-folder`

**Comment by koertho** [comment]
Ich denke, das liegt an unserem Utils-Bundle, das wird den contao templates-ordner noch nicht unterstützen. Da müssten wir hier auch auf das Twig-Support-Bundle umsteigen

**Comment by Olli** [comment]
Ja das liegt vermutlich daran. Ich habe es auch mal nur mit /templates probiert - auch da wird das Template nicht gefunden.
